components/discover/claimModal.tsx

```
@@ -178,25 +147,45 @@ const ClaimModal: FunctionComponent<ClaimModalProps> = ({
           <div className={`${styles.bottomContent} !gap-6 !py-6 !px-5`}>
             <div className="flex w-full justify-between items-center">
               <button onClick={closeModal} aria-label="Cancel claiming rewards">
-                <Typography type={TEXT_TYPE.BODY_MIDDLE}>
-                  Cancel
-                </Typography>
+                <Typography type={TEXT_TYPE.BODY_MIDDLE}>Cancel</Typography>
               </button>
               <div className="w-fit">
-                <Button disabled={claimRewards ? false : true}
+                <Button
+                  disabled={calls ? false : true}
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)